app/controllers/api/v2/rh_cloud/cloud_request_controller.rb

```diff
@@ -44,9 +44,9 @@ def handle_run_playbook_request
       logger.error("Reporting URL is not valid: #{metadata['return_url']}") && return unless valid_url?(metadata['return_url'])
 
       hosts = metadata['hosts'].split(',')
-      host_ids = host_ids(hosts)
+      host_ids = Host.search_for("not params.#{InsightsCloud.enable_cloud_remediations_param} = f").where(id: host_ids(hosts)).pluck(:id)
```

Comment:
could it be break for readability? e.g:
```suggestion
      enabled_hosts = Host.search_for("not params.#{InsightsCloud.enable_cloud_remediations_param} = f")
      host_ids = enabled_hosts.where(id: host_ids(hosts)).pluck(:id)
```